### DISCUSSION TITLE
How do we use a Proxy in Microcks?

### DISCUSSION THREAD

**Discussion by Apoorva64** [microcks/microcks#1087] [Q&A] [answered]
How do we use a Proxy in Microcks?. 
Senario:
i can acces my gitlab instance only though a proxy.

I have read around the source code an found "ProxyConfiguration.java" but i don't know how to use it.

**Comment by lbroudoux** [comment, marked as answer]
Hello,

ProxyConfiguration.java was an attempt to add this at some point but it never get the formal validation of people asking for this feature. It is then not tested and I don't know the current status. We had this docker-compose file here: https://github.com/microcks/microcks/blob/master/install/docker-compose/docker-compose-with-proxy.yml that used to be working at some point. It maybe the good starting point to check how things are going. 

Cheers,

**Reply by Apoorva64** [reply]
Works~
